integration/tests/cook/test_basic.py

```diff
@@ -96,7 +96,6 @@ def test_uid(self):
         finally:
             util.kill_jobs(self.cook_url, [job_uuid], assert_response=False)
 
-    @unittest.skipIf(util.using_kubernetes(), 'Output url is not currently supported on kubernetes')
     def test_output_url(self):
```

Comment:
Add a test cases for reading/downloading/browsing a well-known path outside the sandbox directory; e.g., `/etc/hosts`.